scripts/docker/install-docker.ps1

```diff
@@ -1,102 +1,18 @@
 # https://msdn.microsoft.com/de-de/virtualization/windowscontainers/quick_start/quick_start_windows_10
 Set-ExecutionPolicy Bypass -scope Process
-New-Item -Type Directory -Path 'C:\Program Files\docker\'
-Invoke-WebRequest https://aka.ms/tp5/b/dockerd -OutFile $env:ProgramFiles\docker\dockerd.exe
-Invoke-WebRequest https://aka.ms/tp5/b/docker -OutFile $env:ProgramFiles\docker\docker.exe
-[Environment]::SetEnvironmentVariable("Path", $env:Path + ";C:\Program Files\Docker", [EnvironmentVariableTarget]::Machine)
+Invoke-WebRequest "https://master.dockerproject.org/windows/amd64/docker-1.13.0-dev.zip" -OutFile "$env:TEMP\docker-1.13.0-dev.zip" -UseBasicParsing
```

Comment:
In the meantime I updated the script to 1.12.0 and docker pull, but I believe you that the nightly zip contains more valuable improvements. So this is fine!